ReactAndroid/src/main/java/com/facebook/react/jscexecutor/Android.mk

```diff
@@ -7,12 +7,9 @@ LOCAL_PATH := $(call my-dir)
 
 include $(CLEAR_VARS)
 
-# Flag to enable V8 in react-native code 
-V8_ENABLED := 1
-
 LOCAL_MODULE := jscexecutor
 
-ifeq ($(V8_ENABLED), 0)
+ifeq ($(JS_ENGINEUSED), JSC)
```

Comment:
I don't think we need this check now as we have added check in jni\react\jni\android.mk